aidl-rust-codegen/src/binder_impl/IUListener.rs

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
Note that this is almost completely generated Rust code via the AIDL... not really anything to review!